processor/routingprocessor/factory_test.go

```diff
@@ -31,9 +31,17 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	"go.opentelemetry.io/collector/processor/processorhelper"
 	"go.opentelemetry.io/collector/processor/processortest"
+	"go.opentelemetry.io/otel/metric/noop"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 )
 
+var noopTelemetrySettings = component.TelemetrySettings{
+	TracerProvider: trace.NewNoopTracerProvider(),
```

Comment:
Do we have a sink that we can use to test whether the data points are being created?

Reply:
No, I couldn't find anything like that.